packages/ui-select/src/Select/index.tsx

```diff
@@ -135,15 +136,14 @@ class Select extends Component<SelectProps> {
   private _optionHeight = 36
 
   focus() {
-    this.ref && this.ref.focus()
+    this.inputRef && this.inputRef.focus()
   }
 
   get _input() {
     console.warn(
       '_input property is deprecated and will be removed in v9, please use ref instead'
     )
-
-    return this.ref
+    return this.inputRef
```

Comment:
Technically, if you look back in the git history, this `_input` was doing the exact same thing as the new `inputRef`, so I would just revert it to the original and remove this getter + warning. Also, a name like `inputRef` indicates a component prop that returns with the input element, not a private prop on the component (e.g.: DateInput):

```js
  /**
   * Provides a ref to the underlying input element.
   */
  inputRef?: (element: HTMLInputElement | null) => void
```